.github/workflows/ai-on-demand.yaml

```
@@ -0,0 +1,88 @@
+        fetch-depth: 1
+    
+    - name: Generate Custom App Token
+      id: generate-token
+      uses: actions/create-github-app-token@v1
+      with:
+        app-id: ${{ secrets.AI_APP_ID }}
+        private-key: ${{ secrets.AI_PRIVATE_KEY }}
+    
+    - name: Extract Instruction from Comment
+      id: extract-instruction
+      run: |
```

actionlint (1.7.7)

"github.event.comment.body" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)